src/Monomer/Main/Types.hs

```diff
@@ -385,9 +386,24 @@ appDisableAutoScale disable = def {
 Available fonts to the application. An empty list will make it impossible to
 render text.
 -}
-appFontDef :: Text -> Text -> AppConfig e
-appFontDef name path = def {
-  _apcFonts = [ FontDef name path ]
+appFontDefFile :: Text -> Text -> AppConfig e
+appFontDefFile name path = def {
+  _apcFonts = [ FontDefFile name path ]
+}
+
+{- |
+Alias for 'appFontDefFile' for backwards compatibility.
+-}
+appFontDef = appFontDefFile
+{-# DEPRECATED appFontDef "Use appFontDefFile directly" #-}
```

Comment:
If the changes here get through, I took the liberty to deprecate `appFontDef` in the light of having 2 distinct ways of loading fonts, but that's just a proposal.

Reply:
@klausweiss I'd prefer to keep the existing `appFontDef` name as is, and add the new memory based one as `appFontDefMem`. The main reason is to avoid breaking existing code. A secondary reason is this would make it work similarly to the `image` widget, that has [image](https://github.com/fjvallarino/monomer/blob/main/src/Monomer/Widgets/Singles/Image.hs#L259) for path based images and [imageMem](https://github.com/fjvallarino/monomer/blob/main/src/Monomer/Widgets/Singles/Image.hs#L271) for memory based ones.

If you prefer to keep it as `appFontDefMemory`, it works great for me too.

Reply:
Ok, sounds good. Renamed back to `appFontDefMem`.